runtime/testnet/src/lib.rs

```diff
@@ -1687,18 +1689,22 @@ impl pallet_multi_asset_delegation::Config for Runtime {
 // 	// BridgeAccountNative: 5EYCAe5jLbHcAAMKvLFSXgCTbPrLgBJusvPwfKcaKzuf5X5e
 // 	pub BridgeAccountNative: AccountId32 = SygmaBridgePalletId::get().into_account_truncating();
 // 	// BridgeAccountOtherToken  5EYCAe5jLbHcAAMKvLFiGhk3htXY8jQncbLTDGJQnpnPMAVp
-// 	pub BridgeAccountOtherToken: AccountId32 = SygmaBridgePalletId::get().into_sub_account_truncating(1u32);
-// 	// BridgeAccounts is a list of accounts for holding transferred asset collection
-// 	pub BridgeAccounts: BTreeMap<XcmAssetId, AccountId32> = bridge_accounts_generator();
+// 	pub BridgeAccountOtherToken: AccountId32 =
```

Comment:
@1xstj since we restarted the tesntet can we just remove all this stuff? Please jump into this PR and help test.